lib/organization.js

```diff
@@ -17,18 +17,24 @@ const ensureOrganization = async function (claims) {
     PREFIX org: <http://www.w3.org/ns/org#>
     PREFIX foaf: <http://xmlns.com/foaf/0.1/>
     PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+    PREFIX adms: <http://www.w3.org/ns/adms#>
 
-    SELECT ?organization ?name
+    SELECT ?organization ?name ?status
     WHERE {
       GRAPH <${USERS_GRAPH}> {
         ?organization a foaf:Organization ;
             org:identifier ${sparqlEscapeString(ovoCode)} .
         OPTIONAL { ?organization skos:prefLabel ?name . }
+        OPTIONAL { ?organization adms:status ?status }
       }
     } LIMIT 1`);
 
     if (queryResult.results.bindings.length) {
       const result = queryResult.results.bindings[0];
+      const status = result.status.value;
+      if (status === ACCESS_BLOCKED_STATUS_URI) {
+        throw new Error(`The user's organization has been blocked and does not have access.`)
```

Comment:
Custom error gooien zodat die in app.js opgevangen kan worden en `403 Forbidden` kan returnen.